macaroonbakery/tests/test_bakery.py

```diff
@@ -29,7 +29,7 @@
         }, {
             u'cid': u'allow read-no-terms write'
         }, {
-            u'cid': u'time-before 2016-07-19T14:29:14.312669464Z'
+            u'cid': u'time-before 2158-07-19T14:29:14.312669464Z'
```

Comment:
I'm concerned that you could change this value without changing any of the tests. That implies to me that the client macaroons_expiry_time logic isn't tested at all, which it probably should be. Could we add a test that actually tests the cookie expiry time, please?